syncd/syncd_saiswitch.cpp

```diff
@@ -329,14 +329,12 @@ void SaiSwitch::helperCheckLaneMap()
     }
 
     /*
-     * We assume there are at least 32 ports and port number
-     * can be divided by 16 without rest.
+     * We assume there are at least 32 ports.
      */
 
-    if (laneMap.size() < 32 || (laneMap.size() % 16 != 0))
+    if (laneMap.size() < 32)
```

Comment:
it is interesting to have this check? any reason? if the switch has less than 32 ports, then we print error?

Reply:
what do you mean?

Reply:
I mean why do check the lanemap size at all? there could be switch with less than 32 ports.

Reply:
i added this at first to rule out some errors and mistakes with configuration file since we didnt used any other configuration, but can there be a switch with sonic with 5 ports? :P